benchmarks/nel/scripts/compile_corpora.py

```diff
@@ -0,0 +1,16 @@
+""" Compiles train/dev/test corpora. """
+import typer
+
+from datasets.dataset import Dataset
+
+
+def main(dataset_id: str):
+    """ Create corpora in spaCy format.
+    dataset_id (str): Dataset ID.
+    """
+
+    Dataset.generate_dataset_from_id(dataset_id).compile_corpora()
```

Comment:
Is this creating the `Dataset` again from scratch?

Reply:
Yes. The creation of a `Dataset` instance is very lightweight however - the only thing happening there by default is setting the paths w.r.t. the dataset ID and loading the dataset config file.